### PR TITLE
Test 'crash' utility

### PR DESCRIPTION
**Pull request by mnowaksuse**
FATE#320844

- Not sure where to put this test, choose /toolchain/;
- Not sure what `base` to use, choose what's in /toolchain/ tests;
- Not sure about a proper method to install debuginfo pkgs, choose method from `install.pm` from /toolchain/.

Needles will follow.

### CODE REVIEW

File: products/sle/main.pm [hunk 1]
```diff
@@ -1259,6 +1259,7 @@ if (get_var("STORE_HDD_1") || get_var("PUBLISH_HDD_1")) {
 }
 
 if (get_var("TCM") || check_var("ADDONS", "tcm")) {
+    loadtest "tests/toolchain/crash.pm";
```
Comment (okurz) [inline products/sle/main.pm]: that's wrong, it should say just "toolchain/crash.pm"

File: tests/toolchain/crash.pm [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "opensusebasetest";
+use strict;
+use testapi;
+use utils;
+
+sub run() {
+    my $self = shift;
```
Comment (okurz) [inline tests/toolchain/crash.pm]: blind copy/paste? I don't see you using "self" anywhere

**Comment by okurz** [comment]
nice one.

what about a corresponding TW test? Please check if it works with a current TW snapshot and then add to the main.pm there.

The commit message could have a bit more details, e.g. what issue is fixed, why this needs to change, to which versions of which product it applies, link to a bug or a fate entry, the choices you made, etc. Also see http://chris.beams.io/posts/git-commit/ as a helpful guide how to write good commit messages.

What you put into the description of the pull request could also be in the commit message details.

I can recommend the tool "hub" (zypper in rubygem-hub) for easier PR creation. Also, I myself use a script to create a PR with proper description for these simple one-commit PRs:
```bash
$ cat $(which git-pr-last )
#!/bin/sh -e
target="${target:-"$USER"}"
git push $target && git show --no-patch --format=%B | hub pull-request -F -
```

Please provide a link or screenshot to a local verification run.

**Comment by mnowaksuse** [comment]
SLES Verification run: http://assam.suse.cz/tests/1454.
SLES Needles MR: https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/155 (will likely apply to openSUSE as well).
Blocks openSUSE verification as I am unable to add openSUSE to my openQA instance: https://github.com/os-autoinst/openQA/issues/742.

**Comment by mnowaksuse** [comment]
Leap verification: http://assam.suse.cz/tests/1507
On Tumbleweed it kills the system: http://assam.suse.cz/tests/1510#step/crash/32

**Comment by okurz** [comment]
looks very nice and clean. Let's see where else 'crash' will cause a 'crash' ;-) The "toolchain" might not be the perfect place, I don't know any better and it will not break many other tests if it fails so I don't see this critical.

File: tests/toolchain/crash.pm [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+use utils;
+
+sub run() {
+    select_console('root-console');
+
+    script_run 'zypper -n in yast2-kdump kdump crash';
+
+    script_run 'yast2 kdump', 0;
+
+    assert_screen 'yast2-kdump-disabled';
```
Comment (dzedro) [inline tests/toolchain/crash.pm]: kdumb can be enabled https://openqa.suse.de/tests/446225#step/crash/9

Comment (mnowaksuse) [inline tests/toolchain/crash.pm]: @dzedro https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/1475 ?

**Pull request merged**